modules/library/referencing/src/test/java/org/geotools/referencing/ScriptTest.java

```diff
@@ -314,7 +314,18 @@ public void testSinusoidal() throws Exception {
     public void testGnomonic() throws Exception {
         runScript("scripts/Gnomonic.txt");
     }
-    
+
+    /**
+     * Run "GEOS.txt"
+     *
+     * @throws Exception If a test failed.
+     */
+    @Test
+    @Ignore
```

Comment:
Ooooh, an oversight (and why we do code reviews!).  Initially I copied the last entry from bottom of file, later to realize those were all ignored.  I moved my modified copy behind last unignored test but never removed `@ignored`.  I didn't catch this because I run the script individually.